src/commands/Minion/pc.ts

```diff
@@ -0,0 +1,90 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { BotCommand } from '../../lib/BotCommand';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { requiresMinion, minionNotBusy } from '../../lib/minions/decorators';
+import { Time } from 'oldschooljs/dist/constants';
+import { formatDuration } from '../../lib/util';
+import { rand } from '../../util';
+import { PestControlActivityTaskOptions } from '../../lib/types/minions';
+import { MinigameIDsEnum } from '../../lib/minions/data/minigames';
+import { Activity, Tasks } from '../../lib/constants';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+const landers = [
+	{ name: 'novice', points: 3, gamesRequired: 0, timeToFinish: Time.Minute * 4.5 },
+	{ name: 'intermediate', points: 4, gamesRequired: 100, timeToFinish: Time.Minute * 3.1 },
+	{ name: 'veteran', points: 5, gamesRequired: 200, timeToFinish: Time.Minute * 2 }
```

Comment:
this seems short

Reply:
Which part seems short here?

Average pest control game lasts around 2 minutes on the veteran boat

Reply:
Would store the combat lvl requirments on the landers for future and then make a temp combat variable to 126 which can be replaced in future with the users combat lvl

Reply:
2 mins is just about a perfect game, minions arnt perfect, and considering that this minigame would only really be played once i think increasing the time a bit to an avg of 2.5 min seems reasonable